presto-accumulo/src/main/java/com/facebook/presto/accumulo/index/IndexLookup.java

```diff
@@ -39,13 +41,15 @@
 import org.apache.accumulo.core.security.Authorizations;
 import org.apache.hadoop.io.Text;
 
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map.Entry;
 import java.util.Optional;
 import java.util.Set;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
Build should fail due to wildcarded imports.  Need to import these individually.